sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java

```diff
@@ -33,8 +33,20 @@
 @ScannerSide
 @ServerSide
 public abstract class HttpDownloader extends UriReader.SchemeProcessor {
+
+  /**
+   * Catch-all default timeout, replaced by
+   *   {@link #DEFAULT_READ_TIMEOUT_IN_MILLISECONDS}
+   *   {@link #DEFAULT_CONNECT_TIMEOUT_IN_MILLISECONDS}
+   *
+   * @deprecated since 7.0
+   */
+  @Deprecated
   public static final int TIMEOUT_MILLISECONDS = 20 * 1000;
 
+  public static final int DEFAULT_READ_TIMEOUT_IN_MILLISECONDS = 60 * 1000;
+  public static final int DEFAULT_CONNECT_TIMEOUT_IN_MILLISECONDS = TIMEOUT_MILLISECONDS;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "TIMEOUT_MILLISECONDS"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)